plugin.json

```diff
@@ -109,6 +109,13 @@
                         "value": "disable"
                     }
                 ]
+            },
+            {
+                "key": "EnableWebhookEventLogging",
+                "display_name": "Enable Webhook Event Logging:",
+                "type": "bool",
+                "help_text": "Allow the plugin to log the webhook event. The log level needs to be set to DEBUG.",
+                "default": false
```

Comment:
@dipak-demansol the dafault value is false.

Reply:
> @sibasankarnayak as per discussion with Dylan, it would be best if by default this log is set to false when user install new version on the existing version. because normal user don't need this so every time when he add new version then user don't need to set as false again and again. can you pls add this.

can refer the default value [here](https://github.com/mattermost/mattermost-plugin-github/pull/487/files#r757203300) is already false , if you had made the value as true , it won't set to false again .